demo/src/main/java/com/orange/ods/demo/ui/components/tabs/TopAppBarFixedTabs.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.tabs
+
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.res.painterResource
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.PagerState
+import com.orange.ods.compose.component.tab.OdsLeadingIconTab
+import com.orange.ods.compose.component.tab.OdsTab
+import com.orange.ods.compose.component.tab.OdsTabRow
+import kotlinx.coroutines.launch
+
+@ExperimentalMaterialApi
+@ExperimentalPagerApi
+@Composable
+fun TopAppBarFixedTabs(tabs: List<TabItem>, pagerState: PagerState, tabIconType: SubComponentTabsState.TabIconType, tabTextEnabled: Boolean) {
+    val scope = rememberCoroutineScope()
+
+    OdsTabRow(
+        selectedTabIndex = pagerState.currentPage
+    ) {
+        tabs.forEachIndexed { index, tab ->
```

Comment:
`selected` and `onClick` values can be factorized:

```kotlin
tabs.forEachIndexed { index, tab ->
    val selected = pagerState.currentPage == index
    val onClick: () -> Unit = {
        scope.launch {
            pagerState.animateScrollToPage(index)
        }
    }
    if (tabIconType == SubComponentTabsState.TabIconType.Leading && tabTextEnabled) {
        val icon = painterResource(id = tab.icon)
        val title = tab.title
        OdsLeadingIconTab(icon = icon, text = title, selected = selected, onClick = onClick)
    } else {
        val icon = if (tabIconType == SubComponentTabsState.TabIconType.None) null else painterResource(id = tab.icon)
        val title = if (tabTextEnabled) tab.title else null
        OdsTab(icon = icon, text = title, selected = selected, onClick = onClick)
    }
}
```

Same comment for `TopAppBarScrollableTabs`.